tests/JiraClientStub.ts

```diff
@@ -66,7 +66,7 @@ export const jiraClientStub = {
     console.log(`Invoked jira.moveIssue('${issueId}', '${transitionName}', ${fields})`);
   },
   async transitionIssue(issueId: string, transition: any, fields: any = null): Promise<void> {
-    console.log(`${issueId}: Executing '${transition.name}' (${transition.id}) transition`);
+    console.log(`Invoked jira.transitionIssue('${issueId}', 'transition: ${transition.name} ${transition.id}', ${fields}`);
```

Comment:
If you send something else in that object, I want to see that
```suggestion
console.log(`Invoked jira.transitionIssue('${issueId}', ${JSON.stringify(transition)}, ${JSON.stringify(fields)}`);
```